log_drains.go

```diff
@@ -15,6 +17,15 @@ type LogDrain struct {
 	URL   string `json:"url"`
 }
 
+type errorRes struct {
+	Error string `json:"error"`
+}
+
+type LogDrainRes struct {
```

Comment:
```suggestion
type logDrainRes struct {
```


The `LogDrainRes` structure is never returned. Hence I think it should stay private.